ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -1,9 +1,26 @@
 workflow:
   as: osd-ccs-fleet-manager
   steps:
+    env:
+      HOSTED_CP: "true"
+      ZONES_COUNT: "1"
+      COMPUTE_MACHINE_TYPE: "m5.2xlarge"
+      REPLICAS: "3"
+      DISABLE_WORKLOAD_MONITORING: "false"
+      OCM_LOGIN_ENV: "integration"
+      ROSA_LOGIN_ENV: "integration"
     pre:
     - ref: osd-ccs-fleet-manager-sc-provision
+    - ref: aws-provision-vpc-shared
+    - ref: rosa-sts-account-roles-create
+    - ref: rosa-sts-cluster-provision
+    - ref: rosa-conf-idp-htpasswd
+    - ref: osd-ccs-conf-idp-htpasswd-multi-users
+    - ref: rosa-operators-wait-ready
```

Comment:
Seems like it is better for us to have a chain for provision/deprovision. I will add it tomorrow.

Reply:
After [add rosa-sts chains](https://github.com/openshift/release/pull/38588) is merged, you can update the workflow accordingly.